src/main/java/dk/cs/aau/huppaal/presentations/LogTabPresentation.java

```diff
@@ -0,0 +1,175 @@
+package dk.cs.aau.huppaal.presentations;
+
+import dk.cs.aau.huppaal.BuildConfig;
+import dk.cs.aau.huppaal.HUPPAAL;
+import dk.cs.aau.huppaal.controllers.LogTabController;
+import dk.cs.aau.huppaal.logging.Log;
+import dk.cs.aau.huppaal.logging.LogRegex;
+import dk.cs.aau.huppaal.logging.LogLinkQuantifier;
+import dk.cs.aau.huppaal.presentations.logging.Hyperlink;
+import dk.cs.aau.huppaal.presentations.logging.HyperlinkTextArea;
+import dk.cs.aau.huppaal.presentations.logging.TextStyle;
+import dk.cs.aau.huppaal.presentations.util.PresentationFxmlLoader;
+import dk.cs.aau.huppaal.utility.helpers.SelectHelper;
+import javafx.application.Platform;
+import javafx.beans.NamedArg;
+import javafx.beans.value.ChangeListener;
+import javafx.scene.control.Tooltip;
+import javafx.scene.layout.HBox;
+import org.fxmisc.flowless.VirtualizedScrollPane;
+
+import java.awt.Desktop;
+import java.io.File;
+import java.net.URI;
+import java.util.Optional;
+
+public class LogTabPresentation extends HBox {
+    public LogTabController controller;
+    public boolean autoscroll, wordwrap;
```

Comment:
These values are public, but changing them has no effect on the presentation, and the methods that could be expected to change them are all private. That seems a bit odd. I know that they are changed using the buttons and their values are kept to ensure the correct behavior, but could they not be private?

Reply:
Fixed in 9012755fcec5027a083ff903f53a4d902c540480